Rakefile

```diff
@@ -57,8 +57,8 @@ task 'setup_env' do
    `venv/bin/python venv/ez_setup.py`
    `wget -O venv/get-pip.py https://raw.github.com/pypa/pip/master/contrib/get-pip.py`
    `venv/bin/python venv/get-pip.py`
-   `venv/bin/pip install -r source-requirements.txt`
-   `venv/bin/pip install -r source-optional-requirements.txt`
+   `venv/bin/pip install -r requirements.txt`
+   `PIP_COMMAND=venv/bin/pip ./utils/pip-allow-failures.sh requirements-opt.txt`
```

Comment:
It doesn't work (fails with `Errno::ENOENT: No such file or directory - PIP_COMMAND=venv/bin/pip ./utils/pip-allow-failures.sh requirements-opt.txt`), I managed to make it work with `export PIP_COMMAND=venv/bin/pip && ./utils/pip-allow-failures.sh requirements-opt.txt`. (first arg must be the exec ?)